03_launch_mgmt_cluster.sh

```diff
@@ -131,7 +131,7 @@ function update_images(){
       kustomize edit set image $OLD_IMAGE=$LOCAL_IMAGE
     fi
     declare "$OLD_IMAGE_VAR"="$LOCAL_IMAGE"
-    export OLD_IMAGE_VAR
+    export "$OLD_IMAGE_VAR"
```

Comment:
I think you need {} to pass the shellcheck

Reply:
it's actually the ? that matters to silence the shellcheck. I have modified it